registry/commands.py

```diff
@@ -43,20 +45,20 @@ def install_test_data():
                 import_date,
             ) = tuple(row.values())
 
-            if donation_center not in donation_centers:
-                title = donation_center.replace("_", " ")
-                slug = donation_center.lower()
-                donation_centers[donation_center] = DonationCenter(
-                    title=title, slug=slug
+            if donation_center != "jinde" and donation_center not in donation_centers:
+                # This should not happen, all donation centers should
+                # already be in the database
+                raise RuntimeError(
+                    "Donation center from test data is not present in DB."
                 )
-                db.session.add(donation_centers[donation_center])
-                db.session.commit()
-                db.session.flush()
-                print(donation_centers[donation_center])
+            elif donation_center == "jinde":
+                donation_center_id = None
```

Comment:
Jen zvědavý dotaz 🌼takže když bude v proměnné donation_center_id hodnota None a tato pak bude uložena v tabulce batches ve sloupci donation_center, které je jako datový typ INTEGER, tak jak bude v tabulce tato hodnota pak vypadat? bude tam prázdno? bude tam null? bude tam 0?

Reply:
Správný dotaz! Bude tam NULL, což je databázový ekvivalent Pythoního None a tedy chybějící hodnoty resp. prázdna. V definici tohoto sloupce je totiž napsáno, že tato hodnota chybět může, zatímto sloupec `imported_at` má v definici `NOT NULL`, takže ten musí být vždy vyplněn.